File: APE/SW-AyLP-APE-02: Estructuras de Selección/Códigos_Java/controlHora.java
import java.util.Scanner;

public class controlHora {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int hora, minuto, segundo;
        char respuesta;

        System.out.print("Ingrese la hora: ");
        hora = sc.nextInt();
        System.out.print("Ingrese la minuto: ");
        minuto = sc.nextInt();
        System.out.print("Ingrese la segundos: ");
        segundo = sc.nextInt();

        System.out.println("La hora registrada es: " + hora + ":" + minuto + ":" + segundo);

        System.out.println("¿Desea cambiar la hora? (S/N): ");
        respuesta = sc.next().charAt(0);

        while ((respuesta == 'S')|| respuesta == 's') {
            System.out.print("Ingrese la nueva hora: ");
            hora = sc.nextInt();
            System.out.print("Ingrese los nuevos minutos: ");
            minuto = sc.nextInt();
            System.out.print("Ingrese los nuevos segundos: ");
            segundo = sc.nextInt();

            System.out.println("La nueva hora registrada es: " + hora + ":" + minuto + ":" + segundo);

            System.out.println("¿Desea cambiar la hora nuevamente? (S/N): ");
            respuesta = sc.next().charAt(0);
        }

        System.out.println("Programa Finalizado");
    }
}

File: APE/SW-AyLP-APE-02: Estructuras de Selección/Códigos_C++/controlHora.cpp
# include <iostream>

using namespace std;

int main() {
    int hora, minuto, segundo;
    char respuesta;

    cout << "Ingrese la hora: ";
    cin >> hora;
    cout << "Ingrese los minutos: ";
    cin >> minuto;
    cout << "Ingrese los segundos: ";
    cin >> segundo;

    cout << "La hora registrada es: " << hora << ":" << minuto << ":" << segundo << endl;

    cout << "¿Desea cambiar la hora? (S/N): ";
    cin >> respuesta;

    while (respuesta == 'S' || respuesta == 's'){
        cout << "Ingrese la nueva hora: ";
        cin >> hora;
        cout << "Ingrese los nuevos minutos: ";
        cin >> minuto;
        cout << "Ingrese los nuevos segundos: ";
        cin >> segundo;

        cout << "La nueva hora registrada es: " << hora << ":" << minuto << ":" << segundo << endl;

        cout << "¿Desea cambiar la hora nuevamente? (S/N): ";
        cin >> respuesta;
    }

    cout << "Programa finalizado." << endl;
}